src/diskquota.c

```diff
@@ -784,14 +786,15 @@ disk_quota_launcher_main(Datum main_arg)
 
 		/*
 		 * When db list is empty, curDB is NULL.
-		 * When curDB->in_use is false means dbEtnry has been romoved
-		 * When curDB->dbid doesn't equtal curDBId, it means the slot has
+		 * When curDB->in_use is false means dbEtnry has been removed
```

Comment:
How does this relate to the current patch?

Reply:
Ok, I have removed these obvious mistypings in new commit